From 158bd87b3db04bdd1effe4a92424f9572224a6c3 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Sat, 15 May 2021 00:09:21 +0200
Subject: [PATCH] arm64: dts: ipq8074: add missing reserved memory nodes

Downstream kernel has more reserved memory nodes.
Without these ath11k will cause a board reset.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq8074.dtsi | 36 +++++++++++++++++++++++++--
 1 file changed, 34 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/qcom/ipq8074.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq8074.dtsi
@@ -96,15 +96,47 @@
 			reg = <0x0 0x40000000 0x0 0x01000000>;
 		};
 
-		smem_region: memory@4ab00000 {
+		tzapp: tzapp@4a400000 {
+			no-map;
+			reg = <0x0 0x4a400000 0x0 0x00200000>;
+		};
+
+
+		uboot@4a600000 {
+			no-map;
+			reg = <0x0 0x4a600000 0x0 0x00400000>;
+		};
+
+		sbl@4aa00000 {
+			no-map;
+			reg = <0x0 0x4aa00000 0x0 0x00100000>;
+		};
+
+		smem_region: smem@4ab00000 {
 			no-map;
 			reg = <0x0 0x4ab00000 0x0 0x00100000>;
 		};
 
-		q6_region: memory@4b000000 {
+		tz@4ac00000 {
+			no-map;
+			reg = <0x0 0x4ac00000 0x0 0x00400000>;
+		};
+
+		q6_region: wcnss@4b000000 {
 			no-map;
 			reg = <0x0 0x4b000000 0x0 0x05f00000>;
 		};
+
+		q6_etr_region: q6_etr_dump@50f00000 {
+			no-map;
+			reg = <0x0 0x50f00000 0x0 0x00100000>;
+		};
+
+		m3_dump@51000000 {
+			no-map;
+			reg = <0x0 0x51000000 0x0 0x100000>;
+		};
+
 	};
 
 	firmware {
